main.go

```diff
@@ -94,6 +96,7 @@ func init() {
 	flag.BoolVar(&packages.ValidationDisabled, "disable-package-validation", false, "Disable package content validation.")
 	// The following storage related flags are technical preview and might be removed in the future or renamed
 	flag.BoolVar(&featureStorageIndexer, "feature-storage-indexer", false, "Enable storage indexer to include packages from Package Storage v2 (technical preview).")
+	flag.BoolVar(&featureStorageIndexerLocalDev, "feature-storage-indexer-local-dev", false, "Enable local development setup to test storage indexers (technical preview).")
```

Comment:
Should we use another name for this flag?
Other options that I had in mind were:
- enableStorageIndexerLocal
- storageIndexerDevMode
- storageIndexerTestMode

This flag should also be indicative that it can be used to test other storage indexers like the one proposed in this PoC (SQL storage indexer) https://github.com/elastic/package-registry/pull/1294

Reply:
I like that approach. I'll try it just leaving that `STORAGE_EMULATOR_HOST` environment variable